### PR TITLE
Update right sidebar styling

### LINKED ISSUE #1981: Styling tweaks to cleanup right sidebar
While we work on refactoring the entire site with a new tech stack, we're going to do a quick and simple clean-up of current styling. Just giving the look and feel a little TLC.

### PR DESCRIPTION
**Pull request by brittanystoroz**
# What? Why?
Fixes #1981, cleans up some styling issues on the right sidebar. We're going to do a quick and simple cleanup of some styling on the current site before putting all of our design efforts into the new refactored version.

# Questions
I had some questions about updating the HTML in the 'What's New' section. Locally, I edited `src/main/resources/portal.properties` but I see that there's a rule to ignore that file in the `.gitignore`. Does anybody know which file I should be editing to update the structure of that HTML? Or is this being pulled in from somewhere special that I shouldn't be updating? Specifically, I want the `skin.right_nav.whats_new_blurb` to represent this:

```html
skin.right_nav.whats_new_blurb=<ul><li><a href="news.jsp">New data and features released</a></li><li><a href="tools.jsp">New tools released</a></li></ul> <form action="http://groups.google.com/group/cbioportal-news/boxsubscribe"><label>Sign up for low-volume email news alerts:</label><input type="text" name="email"> <input type="submit" name="sub" value="Subscribe"> </form><p>Or follow us <a href="http://www.twitter.com/cbioportal"><i>@cbioportal</i></a> on Twitter</p>
```

rather than the current HTML, which is:
```html
skin.right_nav.whats_new_blurb=<p> &bull;<a href="news.jsp"> <b>New data and features released</b></a><br/> &bull;<a href="tools.jsp"> <b>New tools released</b></a> </p> <form action="http://groups.google.com/group/cbioportal-news/boxsubscribe"> &nbsp;&nbsp;&nbsp;&nbsp;<b>Sign up for low-volume email news alerts:</b></br> &nbsp;&nbsp;&nbsp;&nbsp;<input type="text" name="email"> <input type="submit" name="sub" value="Subscribe"> </form> &nbsp;&nbsp;&nbsp;&nbsp;<b>Or follow us <a href="http://www.twitter.com/cbioportal"><i>@cbioportal</i></a> on Twitter</b>
```

The restructured markup makes it easier to style, but if this is being used in multiple places/we can't update it for some reason, I can CSS around it.

# Changes proposed in this pull request:
- Removed the gradients in the headers and simplified them to be a little less distracting. We'll remove the gradients from the top navigation bar as well in the near future.
- Fixed up margins & paddings and font sizes

# Screenshots
sidebar-beforeafter

# Notify reviewers
@alisman Tagging you since we talked over the phone a bit about this simple styling refresh we were planning on doing. Most of this PR is just styling changes, but there are a couple of files where I cleaned up some HTML markup to make it easier to work with. Sorry for the big diff on those, let me know if you have trouble reading through them. Is there anyone else who would be good to pull in on reviewing this?  @cBioPortal/frontend

Signed-off-by: Brittany Storoz <brittanystoroz@gmail.com>

### CODE REVIEW

File: portal/src/main/webapp/content/examples.html [hunk 1]
```diff
@@ -1,15 +1,23 @@
-<p>
```
Comment (brittanystoroz) [inline portal/src/main/webapp/content/examples.html]: Changes in this file are just to convert these link tags into an unordered list for easier styling.

**Comment by inodb** [comment]
@brittanystoroz This looks great!

In regards to properties. Those are used to allow for custom styling by different institutions. There is a `properties.EXAMPLE` with default props. There's also some `GlobalProperties` class (forgot the exact name) that contains default props 😫.

Changing those props can be a pain to portal maintainers since they'll have to change them when they update the version. It might be okay if communicated well, but if it's possible to style without changing them it might be better.

**Comment by brittanystoroz** [comment]
@inodb I had a feeling that might be the case. I'll see what I can do with the original HTML, but it might not be the prettiest 😁

**Comment by inodb** [comment]
@brittanystoroz Don't know if you already looked at this as well: https://github.com/cBioPortal/cbioportal/pull/1913. Together with your PR that would really clean up the look significantly.

**Comment by inodb** [comment]
Anyway prolly better to tackle those buttons/header in a separate PR, but just so you know

**Comment by brittanystoroz** [comment]
@inodb yea I took a look at that the other day, thanks! I'm going to rework the header and navigation in my next pull request. Just wanted to keep the PRs tiny, like you said :)

**Comment by brittanystoroz** [comment]
I've added a new commit that leaves the original 'Whats New' HTML in place, and styles around it. Still looks like the screenshot shows, just took a bit of extra CSS to make it work.

This is ready for review. Feedback is appreciated! I'm hoping to make small, incremental changes across the entire site, so by no means is this a finished product, but it's a start :)

**Comment by zhx828** [comment]
I like this, a cleaner design fun here. This is awesome and a great start. Just a minor comment on the header of each section. Should we bold the header or find a way let them stand out? It gets really easy to lose the concentration since most of the parts are having light colors. And I feel like they are in the same section. But this is just my understanding, please down vote me if needed :)

**Comment by brittanystoroz** [comment]
I'm ok with bolding the headings a bit. I can make this change tomorrow.

**Comment by ecerami** [comment]
@brittanystoroz or @alisman:  is there a way to get a URL for us to see all changes in action, e.g. all together on the home page?

**Comment by ecerami** [comment]
@brittanystoroz @jjgao:  I reviewed the pull request locally, and think it looks great!  As per @brittanystoroz's comment above, the goal is to introduce gradual changes to the styling of the portal, in a way that is still compatible with the existing style.  Think design "gradualism".

I nominate @alisman to review code, and if he signs off, I think we are good to merge.  Thanks!

**Comment by pieterlukasse** [comment]
@brittanystoroz looks good :+1: . I have detected one small styling issue in my chrome browser. See screenshot. Maybe you can also remove the gradient from the top menu to make it look consistent with the "subscribe" button? You can quickly achieve this by checking the changes in PR #1913. 

image

**Comment by brittanystoroz** [comment]
@pieterlukasse Hi Peter - the top menu bar styling will come in a subsequent PR. Can you let me know what version of Chrome you're using so I can reproduce the issue locally?

**Comment by pieterlukasse** [comment]
@brittanystoroz I've updated my machine since my last comment. Maybe it is not occurring anymore...but I can't quickly test since the heroku deployment is inactive for this PR.

**Comment by pieterlukasse** [comment]
Ok, I tested locally now. Still same issue:

image

This is my chrome version (on Ubuntu): Version 55.0.2883.87 unknown (64-bit)

**Comment by brittanystoroz** [comment]
Thanks @pieterlukasse, can you try testing again -- I pushed up some CSS changes that should make the heights more rigid. I'm having a hard time reproducing locally (likely because I am running on OSX), but let me know if that issue has been resolved. Screenshots are super helpful! Thanks again.

**Comment by jjgao** [comment]
@brittanystoroz Are you still working on this? I'd like to merge it soon.

**Comment by brittanystoroz** [comment]
@jjgao I am finished with this PR. Was just awaiting verification from @pieterlukasse that the display issue he found had been resolved.

**Pull request merged**